src/languages/rust.js

```diff
@@ -7,6 +7,13 @@ Category: common, system
 */
 
 /** @type LanguageFn */
+
+// ============================================
+// Added to support the r# keyword, which is a raw identifier in Rust.
+const UNDERSCORE_IDENT_RE = '(r#)?[a-zA-Z_]\\w*';
+const IDENT_RE = '(r#)?[a-zA-Z]\\w*';
```

Comment:
```suggestion
const UNDERSCORE_INDENT_RE = '(r#)?[a-zA-Z_]\\w*';
const INDENT_RE = '(r#)?[a-zA-Z]\\w*';
```
fix typo `IDENT` -> `INDENT`

```
const RAW_IDENTIFIER = '(r#)?'
const UNDERSCORE_INDENT_RE = RAW_IDENTIFIER  + hljs.UNDERSCORE_INDENT_RE;
const INDENT_RE = RAW_IDENTIFIER  + hljs.INDENT_RE
```
also, maybe above code looks better? (just suggestion 😄)

Reply:
Thanks for the comment!

That's a great suggestion. I think it looks much better! I'll reflect and commit!